components/__tests__/About.test.tsx

```
@@ -0,0 +1,66 @@
+    string,
+    React.FC<React.HTMLAttributes<HTMLElement> & { children?: React.ReactNode }>
+  > = {}
+  ;['div', 'span', 'h2', 'p'].forEach(tag => {
+    motion[tag] = (
+      props: React.HTMLAttributes<HTMLElement> & { children?: React.ReactNode }
+    ) => React.createElement(tag, props, props.children)
+  })
+  return {
+    motion,
+    useInView: () => true,
+    AnimatePresence: (props: React.PropsWithChildren<{}>) =>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)